azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/parser/V2ConfigParser.java

```diff
@@ -67,7 +67,7 @@ public DockerConfiguration getDockerConfiguration() throws AzureExecutionExcepti
     @Override
     public Runtime getRuntime() throws AzureExecutionException {
         final MavenRuntimeConfig runtime = mojo.getRuntime();
-        if (runtime == null) {
+        if (runtime == null || runtime.isEmpty()) {
```

Comment:
runtime.isEmpty() ? what's that ?

Reply:
We will skip update runtime once user didn't set it. However, maven may auto create the `MavenRuntimeConfig` so that we need to use `isEmpty` to check whether user set the value